### PR TITLE
Reorganized plotting and pipeline

### PR DESCRIPTION
**Pull request by ngreenwald**
Reorganized the figures, updating plotting, and tweaked some of the file names from the pipeline to be more informative.

### CODE REVIEW

File: python_files/nivo_outcomes.py [hunk 1]
```diff
@@ -18,83 +17,15 @@
...
-#                                     timepoint_2_name=pop_2, timepoint_2_list=[pop_2], paired=comparison)
-#     # plot results
-#     output_dir = plot_dir + '/evolution_{}'.format(comparison)
-#     if not os.path.exists(output_dir):
-#         os.makedirs(output_dir)
-#
-#     summarize_timepoint_enrichment(input_df=compare_df, feature_df=timepoint_features, timepoints=[pop_1, pop_2],
-#                                  pval_thresh=2, diff_thresh=0.3, output_dir=output_dir)
+# load previously computed results
+combined_df = pd.read_csv(os.path.join(base_dir, 'analysis_files/timepoint_combined_features.csv'))
```
Comment (ngreenwald) [inline python_files/nivo_outcomes.py:28]: I renamed `combined_df.csv` to `timepoint_combined_features.csv` so the filename is more informative. I think I made all the substitutions, but not sure.

File: python_files/nivo_outcomes.py [hunk 2]
```diff
@@ -139,24 +70,24 @@
 #                                         variable_col='Time_to_progression_weeks_RECIST1.1', output_dir=current_plot_dir, min_score=0.95)
 
 # summarize hits from all comparisons
-total_dfs = pd.concat(total_dfs)
-total_dfs['log10_qval'] = -np.log10(total_dfs.fdr_pval)
+ranked_features_df = pd.concat(total_dfs)
```
Comment (ngreenwald) [inline python_files/nivo_outcomes.py:73]: I kept `total_dfs` as the list accumulator, but once they're combined together I switched the name to ranked_features_df to be more informative. Same with the output file name.

File: python_files/postprocessing/7_create_evolution_df.py [hunk 3]
```diff
@@ -44,7 +44,7 @@
     evolution_dfs.append(evolution_df_wide)
 
 evolution_df = pd.concat(evolution_dfs)
-evolution_df.to_csv(os.path.join(output_dir, 'nivo_outcomes_evolution_df.csv'), index=False)
+evolution_df.to_csv(os.path.join(output_dir, 'timepoint_evolution_features.csv'), index=False)
```
Comment (ngreenwald) [inline python_files/postprocessing/7_create_evolution_df.py:47]: Renamed this to be more clear as well

File: python_files/postprocessing/7_create_evolution_df.py [hunk 4]
```diff
@@ -71,4 +71,4 @@
                                                     'Patient_ID', 'Timepoint', 'Induction_treatment', 'Time_to_progression_weeks_RECIST1.1', 'Censoring_PFS_RECIST1.1', 'Clinical_benefit']]])
 combined_df['combined_name'] = combined_df.feature_name_unique + '__' + combined_df.Timepoint
 
-combined_df.to_csv(os.path.join(analysis_dir, 'nivo_outcomes_combined_df.csv'), index=False)
+combined_df.to_csv(os.path.join(analysis_dir, 'timepoint_combined_features.csv'), index=False)
```
Comment (ngreenwald) [inline python_files/postprocessing/7_create_evolution_df.py:74]: This is now the primary input file

**Review by ngreenwald — COMMENTED** [review]
@camisowers see below for highlights of the key points

File: python_files/postprocessing/7_create_evolution_df.py (hunk 3 above)
Comment (camisowers) [reply python_files/postprocessing/7_create_evolution_df.py:47]: Need to also change the file name read in on line 62 below.

File: python_files/postprocessing/7_create_evolution_df.py (hunk 4 above)
Comment (camisowers) [reply python_files/postprocessing/7_create_evolution_df.py:74]: Need to change this file name in the readme.

**Review by camisowers — CHANGES_REQUESTED** [review]
Two small additions, but other than that it all looks good.

**Comment by ngreenwald** [comment]
Good catches, thanks

**Pull request merged**